include/bm_configs_generic.h

```diff
@@ -29,7 +29,7 @@ typedef struct {
 } __attribute__((packed, aligned(1))) GenericConfigKey;
 
 const GenericConfigKey *
-bcmp_config_get_stored_keys(uint8_t &num_stored_keys,
+bcmp_config_get_stored_keys(uint8_t *num_stored_keys,
```

Comment:
This is C++ only syntax and could not be used when compiling the configs.c file.